### PR TITLE
Bug fixed position on menu in text field

### LINKED ISSUE #6394: [UI] Options list overlaps the input box.
#### Current Behavior
[UI] Options list overlaps the input box.
Screenshot 2022-10-17 010603

#### Desired Behavior
Screenshot 2022-10-17 010516

#### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Udit-takkar**
Signed-off-by: Udit Takkar <udit.07814802719@cse.mait.ac.in>

This PR fixes #6394

Screenshot 2022-11-19 at 7 11 33 PM

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thank you for this contribution! Please note pull requests that have changes that are non-material to the issue at-hand slow down the review process as maintainers examine and consider each changed line of code. Often, code beautification tools / formatters will inadvertently make any number of other changes, causing the review cycle to slow down. Please help project maintainers merge your PRs as quickly as possible by only including changes that are pertinent to the specific issue.

**Comment by Nikhil-Ladha** [comment]
Adding to the above point, please only work on PRs once you have the issue assigned to yourself it helps avoid unnecessary conflicts/duplicate work. I see you have neither requested for assignment nor have the issue assigned to you but you sent out a PR. Please refrain from doing so in the future.

**Comment by Udit-takkar** [comment]
@leecalcote actually i raised the issue here about formatting and how should.i fix it https://layer5io.slack.com/archives/C023QGMSH8D/p1668866887309029.  prettier was making some change in code formatting in this PR which i have disabled but still using lint:fix not formatting like in the master branch. ( this is not a problem in my other PRs because i disabled the extensions).

File: ui/components/MesheryPlayComponent.js [hunk 1]
```diff
@@ -13,50 +11,40 @@ import { bindActionCreators } from "redux";
 import { setAdapter } from "../lib/store";
 
 const styles = (theme) => ({
-  icon : { fontSize : 20, },
-  playRoot : { padding : theme.spacing(0),
-    marginBottom : theme.spacing(2), },
-  buttons : { display : "flex",
```
Comment (Udit-takkar) [inline ui/components/MesheryPlayComponent.js:19]: Also i think there are also some problems in some files like this as you can see playroot and margin bottom are not vertically aligned in the master branch as they should after formatting.

**Comment by leecalcote** [comment]
> @leecalcote actually i raised the issue here about formatting and how should.i fix it https://layer5io.slack.com/archives/C023QGMSH8D/p1668866887309029. prettier was making some change in code formatting in this PR which i have disabled but still using lint:fix not formatting like in the master branch. ( this is not a problem in my other PRs because i disabled the extensions).

Oh, that’s great.

**Comment by leecalcote** [comment]
@Harshit-Dandriyal feedback that you might offer here?

**Comment by Harshit-Dandriyal** [comment]
nice work @Udit-takkar , I am just these some extra linting changes, was done by you, or happened automatically?

**Comment by Udit-takkar** [comment]
@Harshit-Dandriyal it happened automatically. I even tried eslint --fix but got the same formatting.

**Comment by Harshit-Dandriyal** [comment]
we are good to go then we can merge this @leecalcote , LGTM

**Review by Harshit-Dandriyal — APPROVED** [review]

**Review by leecalcote — APPROVED** [review]
Thanks, @Udit-takkar and @Harshit-Dandriyal

**Pull request merged**